components/shared/DayRangePicker.tsx

```diff
@@ -41,7 +41,7 @@ export const DayRangePicker = ({ onDayRangeChanged }: DayRangePickerProps) => {
 
   return (
     <label className="w-max font-semibold">
-      <span className="sr-only">Period:</span>
+      <span className="sr-only">Range:</span>
```

Comment:
This is so assistive technologies can read the label. As far as I know, the `insetLabel` is `::before` content which won't get picked up by assistive technologies.